Tools/Scripts/webkitpy/api_tests/manager.py

```diff
@@ -150,8 +151,11 @@ def _print_tests_result_with_status(self, status, runner):
                 self._stream.writeln('')
 
     def _initialize_devices(self):
-        if 'simulator' in self._port.port_name:
-            SimulatedDeviceManager.initialize_devices(DeviceRequest(self._port.DEVICE_TYPE, allow_incomplete_match=True), self.host, simulator_ui=False)
+        # FIXME: Remove the special-case for iOS Simulator once https://bugs.webkit.org/show_bug.cgi?id=256806 is resolved
+        if self._port.port_name in ('iphone-simulator', 'ios-simulator'):
+            SimulatedDeviceManager.initialize_devices(DeviceRequest(DeviceType(hardware_family='iPhone', hardware_type='SE (3rd generation)'), allow_incomplete_match=True), self.host, simulator_ui=False)
+        elif 'simulator' in self._port.port_name:
```

Comment:
Are there any cases where this differs to `.endswith('-simulator')`?

Reply:
Not at the moment, but I think `in` more accurately reflects the sentiment here. If someone were to do something like `ios-simulator-legacy` (which would seem a perfectly reasonable port name), `ends with('-simulator')` here would break that class name.